src/Sentry.AspNetCore/IUserFactory.cs

```diff
@@ -3,7 +3,13 @@
 namespace Sentry.AspNetCore;
 
 /// <summary>
+/// <para>
 /// Sentry User Factory
+/// </para>
+/// <para>
+/// Note: This interface is tightly coupled to AspNetCore and Will be removed in version 4.0.0. Please consider using
```

Comment:
If we plan to remove it on 4.0.0, should this be marked as `[Obsolete]`?
You can `#pragma` ignore the usage on `DefaultUserFactory` until we get rid of it

Reply:
Done.